.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "env": {
+    "es6": true
+  },
+  "extends": ["plugin:prettier/recommended"],
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "plugins": ["prettier"],
+  "rules": {
+    "object-curly-spacing": ["error", "always"],
```

Comment:
I believe prettier's bracket spacing takes care of this so we should never hit this error